src/tests/test_blindsig.py

```diff
@@ -1,52 +1,69 @@
 """
-Test for ECC blind signatures
+Tests using API.
 """
-import os
+
 import unittest
 from ctypes import cast, c_char_p
-
-from pybitmessage.pyelliptic.eccblind import ECCBlind
-from pybitmessage.pyelliptic.openssl import OpenSSL
+import sys
+from datetime import datetime
+from datetime import timedelta
+sys.path.insert(1, '/home/cis/ektarepo/PyBitmessage/src/pyelliptic')
+from eccblindChain import ECCBlindChain
+from eccblind import ECCBlind
+from openssl import OpenSSL
+EXP = datetime.now() + (timedelta(days=365))
+SIGNER_OBJ_LIST = []
+MSG_UNPACK_CHAIN = []
 
 
 class TestBlindSig(unittest.TestCase):
     """
-    Test case for ECC blind signature
+        Class to test blind signature mechanism
     """
     def test_blind_sig(self):
-        """Test full sequence using a random certifier key and a random message"""
-        # See page 127 of the paper
-        # (1) Initialization
-        signer_obj = ECCBlind()
-        point_r = signer_obj.signer_init()
-
-        # (2) Request
-        requester_obj = ECCBlind(pubkey=signer_obj.pubkey)
-        # only 64 byte messages are planned to be used in Bitmessage
-        msg = os.urandom(64)
-        msg_blinded = requester_obj.create_signing_request(point_r, msg)
-
-        # check
-        msg_blinded_str = OpenSSL.malloc(0, OpenSSL.BN_num_bytes(msg_blinded))
-        OpenSSL.BN_bn2bin(msg_blinded, msg_blinded_str)
-        self.assertNotEqual(msg, cast(msg_blinded_str, c_char_p).value)
-
-        # (3) Signature Generation
-        signature_blinded = signer_obj.blind_sign(msg_blinded)
+        callchain = ECCBlindChain()
```

Comment:
chain test should be a separate one. Leave the existing test as it is and write a new one, e.g.

 `def test_blind_sig_chain(self):`